api/src/main/java/com/spotify/ffwd/model/v1/Metric.java

```diff
@@ -0,0 +1,87 @@
+/*-
+ * -\-\-
+ * FastForward API
+ * --
+ * Copyright (C) 2016 - 2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.ffwd.model.v1;
+
+
+
+
+import com.google.common.base.Charsets;
+import com.google.common.hash.HashFunction;
+import com.google.common.hash.Hasher;
+import com.google.common.hash.Hashing;
+import com.spotify.ffwd.model.v1.Batch;
+import com.spotify.ffwd.v1.Value;
+import java.util.Date;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeMap;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+/**
+ * TODO: rename to 'Point' to follow along with internal guidelines.
+ */
+@Data
+@EqualsAndHashCode(of = {"key", "riemannTags", "tags"})
```

Comment:
Don't we need to use resource tags here? @lmuhlha 
I think riemannTags should be removed as we are not using those anymore.